corehq/apps/reports/standard/deployments.py

```diff
@@ -139,6 +141,22 @@ def rows(self):
             )
         return rows
 
+    @property
+    def export_table(self):
+        def _fmt_ordinal(ordinal):
+            if ordinal >= 0:
```

Comment:
(not 100% sure we should be relying on this behavior but it doesn't need to block)